direct.go

```diff
@@ -25,6 +27,8 @@ const (
 	defaultMaxAllowedCachedAge = 24 * time.Hour
 	defaultMaxCacheSize        = 1000
 	defaultCacheSaveInterval   = 5 * time.Second
+	maxTries                   = 6
+	testURL                    = "http://d157vud77ygy87.cloudfront.net/measurements" // borda
```

Comment:
Okay, I've added a /ping endpoint on borda that does basically nothing. I suspect borda will be able to handle the load just fine, but we'll keep an eye on it.

The alternative would be to set up some other service that can handle posts, but that's just one more thing to maintain.